### PR TITLE
add PyTorchLightning icon

### PR DESCRIPTION
**Pull request by Borda**
simpleicons

**Issue:**
**GitHub stars:** 13.400
**Alexa rank:** 308,918
  
https://www.alexa.com/siteinfo/pytorch-lightning.readthedocs.io

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

adding deep learning framework https://pytorchlightning.ai/

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Welcome to Simple Icons, @Borda, and thanks for the contribution 🙂

I'm noticing some small differences around the outer edges ([example](https://user-images.githubusercontent.com/15157491/118117445-8aa5c580-b3e3-11eb-996f-35ba57566653.png), with yours in red) due to your version being slightly wider than the source. Could you have a another look to see if you can fixe that up, please?

Also, could you update the source URL to https://github.com/PyTorchLightning/pytorch-lightning/blob/a584196abf820179adb0758ef67ddae91c44e7bc/docs/source/_static/images/icon.svg

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7040,6 +7040,11 @@
             "source": "https://github.com/pytorch/pytorch.github.io/blob/8f083bd12192ca12d5e1c1f3d236f4831d823d8f/assets/images/logo.svg",
             "guidelines": "https://github.com/pytorch/pytorch.github.io/blob/381117ec296f002b2de475402ef29cca6c55e209/assets/brand-guidelines/PyTorch-Brand-Guidelines.pdf"
         },
+        {
+            "title": "PyTorch Lightning",
+            "hex": "792EE5",
+            "source": "https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/b181b8c646ace2a23c8ffe10cd4aa0d36af202fa/docs/source/_static/images/icon.svg"
```
Comment (Borda) [inline _data/simple-icons.json:7046]:
```suggestion
            "source": "https://github.com/PyTorchLightning/pytorch-lightning/blob/a584196abf820179adb0758ef67ddae91c44e7bc/docs/source/_static/images/icon.svg"
```

**Comment by Borda** [comment]
@PeterShaggyNoble edited the link...

File: icons/pytorchlightning.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>PyTorch Lightning icon</title><path d="M12 0L1.736 6v12L12 24l10.264-6V6zm-1.777 18l1.08-4.657-2.43-2.397L13.792 6l-1.083 4.665 2.417 2.384z"/></svg>
```
Comment (Borda) [inline icons/pytorchlightning.svg:1]: @PeterShaggyNoble talking about the small differences around the outer edges, do you see in the illustration or the this attached svg?

Comment (PeterShaggyNoble) [reply icons/pytorchlightning.svg:1]: @Borda, the screenshot I linked to above is of the source SVG (in black) overlaid with the version in the PR (in red). After resizing it to `24` high, the source SVG is `20.5` wide whereas the PR version is `20.528` wide.

Comment (Borda) [reply icons/pytorchlightning.svg:1]: I see now, so I ll update the linked icon and come back... :]

Comment (PeterShaggyNoble) [reply icons/pytorchlightning.svg:1]: Hold that thought, @Borda, didn't notice you'd updated the SVG in the PR - let me do another comparison ...

Comment (Borda) [reply icons/pytorchlightning.svg]: ok, made the correct scaling, thx for catching this issue :]

**Review by PeterShaggyNoble — APPROVED** [review]
That looks perfect now, @Borda; nice work 👍🏻

**Pull request merged**